src/cosalib/qemuvariants.py

```diff
@@ -91,6 +91,10 @@
             "--format=oldgnu"
         ]
     },
+    "kubevirt": {
+        "image_format": "qcow2",
+        "platform": "kubevirt",
```

Comment:
Should this be `metal` instead? E.g. are you looking to actually add new platform logic for Ignition, or does kubevirt provisioning happen some other way?